docs/installation/sources/sources_mac.rst

```diff
@@ -0,0 +1,370 @@
+The Xcode command line tools package is separate from Xcode and allows for command line development in mac.
+The previous step should have installed Xcode CLI, to check the correct installation run the following command:
+
+.. code-block::
+
+        gcc --version
+
+.. _cmake_gcc_pip3_wget_git_sm:
+
+CMake, g++, pip3, wget and git
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These packages provide the tools required to install *eProsima Fast DDS* and its dependencies from command line.
+Install CMake_, pip3_ and wget_ using the Homebrew package manager:
+
+.. code-block:: bash
+
+        brew install cmake python3 wget
+
+.. _gtest_sm:
+
+Gtest
+^^^^^
+
+GTest is a unit testing library for C++.
+By default, *eProsima Fast DDS* does not compile tests.
+It is possible to activate them with the opportune
+`CMake configuration options <https://cmake.org/cmake/help/v3.6/manual/cmake.1.html#options>`_
+when calling colcon_ or CMake_.
+For more details, please refer to the :ref:`cmake_options` section.
+For a detailed description of the Gtest installation process, please refer to the
+`Gtest Installation Guide <https://github.com/google/googletest>`_.
+
+
+.. _dependencies_sm:
+
+Dependencies
+------------
+
+*eProsima Fast DDS* has the following dependencies, when installed from binaries in a Linux environment:
+
+* :ref:`asiotinyxml2_sm`
+* :ref:`openssl_sm`
+
+.. _asiotinyxml2_sm:
+
+Asio and TinyXML2 libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Asio is a cross-platform C++ library for network and low-level I/O programming, which provides a consistent
+asynchronous model.
+TinyXML2 is a simple, small and efficient C++ XML parser.
+Install these libraries using Homebrew:
+
+.. code-block:: bash
+
+        brew install asio tinyxml2
+
+.. _openssl_sm:
+
+OpenSSL
+^^^^^^^
+
+OpenSSL is a robust toolkit for the TLS and SSL protocols and a general-purpose cryptography library.
+Install OpenSSL_ using Homebrew:
+
+.. code-block:: bash
+
+        brew install openssl@1.1
+
+
+.. _colcon_installation_mac:
+
+Colcon installation
+-------------------
+
+colcon_ is a command line tool based on CMake_ aimed at building sets of software packages.
+This section explains how to use it to compile *eProsima Fast DDS* and its dependencies.
+
+#. Install the ROS 2 development tools (colcon_ and vcstool_) by executing the following command:
+
+   .. code-block:: bash
+
+       pip3 install -U colcon-common-extensions vcstool
+
+
+#. Create a :code:`Fast-DDS` directory and download the repos file that will be used to install
+   *eProsima Fast DDS* and its dependencies:
+
+   .. code-block:: bash
+
+       mkdir ~/Fast-DDS
+       cd ~/Fast-DDS
+       wget https://raw.githubusercontent.com/eProsima/Fast-DDS/master/fastrtps.repos
+       mkdir src
+       vcs import src < fastrtps.repos
+
+#. Build the packages:
+
+   .. code-block:: bash
+
+       colcon build --cmake-args -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl -DOPENSSL_LIBRARIES=/usr/local/opt/openssl/lib
```

Comment:
```suggestion
       colcon build
```
`colcon build` works fine. OpenSSL is only needed when building with SECURITY. I would state this in the note, but not include those options generally.